test/integration/models/test_networking.py

```diff
@@ -16,36 +16,29 @@ def test_get_networking_rules(get_client, create_firewall):
 
 
 @pytest.mark.smoke
-def test_ip_addresses_share(self):
+def test_ip_addresses_share(get_client):
     """
     Test that you can share IP addresses with Linode.
     """
-    ip_share_url = "/networking/ips/share"
+    client = get_client
+    available_regions = client.regions()
+    chosen_region = available_regions[0]
+    label = "test-ip-share"
+
+    linode_instance, password = client.linode.instance_create(
```

Comment:
Could we move this test instance to a fixture so we can guarantee it will be cleaned up on test failures?

Reply:
Yes, moved into fixtures to create linode instances.